fetch/client/src/main/scala/endpoints4s/fetch/EndpointsSettings.scala

```diff
@@ -0,0 +1,3 @@
+package endpoints4s.fetch
+
+case class EndpointsSettings(host: Option[String])
```

Comment:
I suggest we design it to be “evolvable” from the ground start. This means, it has to be a plain old class like this one for example:

https://github.com/endpoints4s/endpoints4s/blob/0ecaa1d861f631465fc797d195b91372f009cf54/algebras/algebra/src/main/scala/endpoints4s/algebra/Endpoints.scala#L64-L134